src/libraries/Common/tests/System/DirectoryServices/LdapConfiguration.cs

```diff
@@ -77,77 +80,95 @@ internal bool IsActiveDirectoryServer
         internal static LdapConfiguration GetConfiguration(string configFile)
         {
             if (!File.Exists(configFile))
+                Environment.FailFast("LDAP test configuration file not found: " + configFile);
```

Comment:
> Environment.FailFast("LDAP test c [](start = 15, length = 34)

Why we fail fast the tests when there is no provided configuration file. Sorry if I missed in earlier comments but will try to read the whole thread.

Reply:
The config file is copied by the build: it should always be present. I had an error earlier where it wasn't, and the tests don't fail, just skip, which is less obvious. I can revert that if you want but I figured it was helpful

Reply:
I am fine with what you have. Let's see how it'll goes.